Added computeFUV function

francislaus opened the pull request:
Added the function computeFUV which computes the vector FUV by analyzing
the matrix PUM and the vector PUV. Also added a test function for this
functionality.

Solve #17

francislaus committed:
Added computeFUV function

Added the function computeFUV which computes the vector FUV by analyzing
the matrix PUM and the vector PUV. Also added a test function for this
functionality.

apeinot commented:
The function and tests solve well #17. Like in the pull request #22, please add a small comment at the beginning of the test (cf. https://www.monperrus.net/martin/introduction-unit-tests)

Jonagoo commented:
The function looks good, however the documentation for computeFUV reads "computePUM computes..."

francislaus committed:
Added test description for #17

Added a description for the testComputeFUV function, which tests the
function computeFUV of Decide.java

francislaus committed:
Improved description of function computeFUV for #17

Improved the wording in the description of the computeFUV function

francislaus committed:
Improved description of function computeFUV again for #17

Changed the description of function computeFUV so that it describes
computeFUV in a better way

apeinot commented:
The fix you did resolve well the things point out in the reviews. I will resolve the conflicts and merge the branch to resolve #17

apeinot69 committed:
Merge branch 'master' into issue17

Resolve conflicts with master in order to resolve #17
